### DISCUSSION TITLE
Use with nextjs13

### DISCUSSION THREAD

**Discussion by victormongi** [tailwindlabs/headlessui#2187] [Help]
Hello guys I need help,
how to solve this error
I just use normal install 
but got this:
<img width="1171" alt="Screenshot 2023-01-16 at 2 56 57 PM" src="https://user-images.githubusercontent.com/5617746/212616380-8306432b-e18f-40c5-9e47-15325036e1d5.png">

<img width="1179" alt="Screenshot 2023-01-16 at 2 57 25 PM" src="https://user-images.githubusercontent.com/5617746/212616451-c7194386-9844-4208-b844-91ae90e12d01.png">

thank you

**Comment by 88Navistar** [comment]
Add the string 'use-client' to the top of your navigation file
